mirage-conduit.opam

```diff
@@ -7,19 +7,18 @@ bug-reports:  "https://github.com/mirage/ocaml-conduit/issues"
 tags:         "org:mirage"
 license:      "ISC"
 
-build:   ["ocamlfind" "query" "conduit.mirage"]
+build: [
+  ["jbuilder" "subst" "-p" name "--name" name] {pinned}
```

Comment:
Not sure you need to repeat `-p` and `-n` here. `["jbuilder" "subst" "-n" name]` should be enough.